BlazarUI/app/scripts/actions/buildActions.js

```diff
@@ -34,7 +34,7 @@ function getModule() {
   const modulesPromise = branchModules.fetch();
 
   modulesPromise.done( () => {
-    gitInfo.moduleId = _.find(branchModules.data, (m) => {
+    gitInfo.moduleId = find(branchModules.data, (m) => {
```

Comment:
ah gotcha. yea I'm guessing we don't need to be too worried about IE given Blazar's typical user.